src/protocols/rdp/rdp_settings.c

```diff
@@ -356,6 +357,11 @@ enum RDP_ARGS_IDX {
      */
     IDX_PRECONNECTION_BLOB,
 
+    /**
+     * The timezone to pass through to the RDP connection.
```

Comment:
What format are these timezones in? Is there some reference users can consult to determine what legal values are available?

Reply:
Added reference to Wikipedia page.